inputs/sample_input.txt

```diff
@@ -0,0 +1,81 @@
+Marlie nearly turned around when Reed stepped out of the souped-up armored truck, pounding an extra-large energy drink. This was not her scene, but she couldn't trust anyone else to deploy her drone system in the middle of the storm. If she wanted to finish her PhD thesis sometime in the next decade, she'd have to ride shotgun with this adrenaline-junkie and his video camera.
```

Comment:
Change sample to three_little_pigs.

And if keeping another one, make it the goats one.